src/main/kotlin/team/mozu/dsm/global/error/exception/ErrorCode.kt

```diff
@@ -11,6 +11,11 @@ enum class ErrorCode(
     EXPIRED_TOKEN(HttpStatus.UNAUTHORIZED, "Expired Token"),
     INVALID_TOKEN(HttpStatus.UNAUTHORIZED, "Invalid Token"),
 
+    // s3
+    IMAGE_NOT_FOUND(HttpStatus.NOT_FOUND, "Image Not Found"),
+    FAILED_UPLOAD(HttpStatus.INTERNAL_SERVER_ERROR, "Failed_Upload"),
+    FAILED_DELETE(HttpStatus.INTERNAL_SERVER_ERROR, "Failed_Delete"),
```

Comment:
언더바보다는 공백을 사용하는 것이 더 좋을 것 같고, 아래와 같이 좀 더 구체적인 메시지를 작성하는 것이 좋을 것 같습니다
```suggestion
    IMAGE_UPLOAD_FAILED(HttpStatus.INTERNAL_SERVER_ERROR, "Image Upload Failed"),
    IMAGE_DELETE_FAILED(HttpStatus.INTERNAL_SERVER_ERROR, "Image Delete Failed"),
```

Reply:
2f04d0118bd28c5d5f374b9d74eff2047d3a86cf